dags/iop/parser.py

```diff
@@ -239,16 +241,21 @@ def _get_license(self, article: ET.Element):
         return licenses
 
     def _get_publication_info(self, article):
+        journal_title = article.find(
+            "front/journal-meta/journal-title-group/journal-title",
+        )
+        journal_volume = article.find(
+            "front/article-meta/volume",
+        )
+        journal_year = self._get_journal_year(article)
+        journal_issue = article.find("front/article-meta/issue")
+        journal_artid = article.find("front/article-meta/elocation-id")
         return [
             {
-                "journal_title": article.find(
-                    "front/journal-meta/journal-title-group/journal-title",
-                ).text,
-                "journal_volume": article.find(
-                    "front/article-meta/volume",
-                ).text,
-                "journal_year": self._get_journal_year(article),
-                "journal_issue": article.find("front/article-meta/issue").text,
-                "journal_artid": article.find("front/article-meta/elocation-id").text,
+                "journal_title": journal_title is not None and journal_title.text,
```

Comment:
Similar for below ones:

```suggestion
                "journal_title": journal_title.text if journal_title else None,
```

Reply:
same as before